src/libraries/System.Diagnostics.Process/tests/ProcessModuleTests.cs

```diff
@@ -68,5 +68,24 @@ public class ModuleCollectionSubClass : ProcessModuleCollection
         {
             public ModuleCollectionSubClass() : base() { }
         }
+
+        [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
+        public void ModulesAreDisposedWhenProcessIsDisposed()
+        {
+            Process process = CreateDefaultProcess();
+
+            ProcessModuleCollection modulesCollection = process.Modules;
```

Comment:
Do we generally require RemoteExecutor tests to be marked outerloop? Or is that not the case?

Reply:
afaik it's not the case for Process tests that use `CreateDefaultProcess()` method which internally uses `RemoteExecutor`